src/pixi/loaders/ResourceLoader.hx

```diff
@@ -130,7 +130,7 @@ extern class ResourceLoader extends EventEmitter {
 	 * @param [callback] {function} Optional callback that will be bound to the `complete` event.
 	 * @return {Loader}
 	 */
-	function load(?cb:Dynamic):ResourceLoader;
```

Comment:
callback can't be `Void->Void` as it can have 2 parameters. Loader itself and resources object.